### DISCUSSION TITLE
框架在哪里下载

### DISCUSSION THREAD

**Discussion by 447275652** [mamoe/mirai#713] [Q&A] [answered]
求解 框架在哪下载

**Comment by Him188** [comment]
https://github.com/mamoe/mirai/pull/710/files?short_path=422bc79#diff-422bc7940f422986847e7cc49aca3f9f77ff61a2b8554791d950d88c97c22af8

**Comment by Him188** [comment, marked as answer]
https://github.com/mamoe/mirai/blob/dev/docs/ConfiguringProjects.md#d-%E4%B8%8B%E8%BD%BD-jar-%E5%8C%85
